### ISSUE TITLE
gitbook not auth

### ISSUE THREAD

**Issue by sky5454** [lersi/edk2_project_template#25]

**Comment by lersi** [comment]
Hello, I do not understand.  
can you please explain?

**Comment by sky5454** [comment]
https://app.gitbook.com/o/-M5spcURlKaXB8ydA2fr/c/-M5spcW4XcBAejCSbSIN/
open  the link and login with my  github account, but see:
![Screenshot_2024_0424_015029](https://github.com/lersi/edk2_project_template/assets/12370628/4fe4508a-2446-4d29-8ef0-b0152e30f864)

**Comment by lersi** [comment]
You are right, it appears that the gitbook links are broken, they worked at the day of writing, I will update them.
